core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/Printer.java

```diff
@@ -96,4 +96,14 @@ public void setName(String name) {
     this.name = name;
   }
 
+  @Override
+  public String getDeleteType() {
+    return "Printer";
+  }
+
+  @Override
+  public String getDeleteDescription() {
+    return getName() + " (" + getDriver().name() + ")";
+  }
```

Comment:
Probably useful to see `getName() + " (" + getDriver().name() + "/" + getBackend().name() + ")"`